CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ INTERNAL NOTE
 ### [3.29.3](https://github.com/flutter/flutter/releases/tag/3.29.3)
 - [flutter/165166](https://github.com/flutter/flutter/pull/165166) - Impeller,
   All platforms, Text that is scaled over 48x renders incorrectly.
+- [flutter/161466](https://github.com/flutter/flutter/pull/161466) - Fixed issue where hot restart could hang indefinitely if "Pause on Unhandled Exceptions" was enabled and a call to `Isolate.run` had not completed.
```

Comment:
@bkonyi This doesn't follow our changelog guidance. 

https://github.com/flutter/flutter/blob/master/docs/releases/Hotfix-Documentation-Best-Practices.md

Here is my best attempt at the entry. 
```suggestion
- [flutter/161466](https://github.com/flutter/flutter/pull/161466) -When "Pause on Unhandled Exceptions" is enabled in VsCode hot restart can hang on all platforms. 
```

Reply:
To be fair, there's a lot of entries in recent change logs that I don't understand as a Flutter developer... 😄

I think this is the simplest the description can be without losing important context.